### ISSUE TITLE
Add tests on the agglayer CLI

### ISSUE THREAD

**Issue by Freyskeyd** [agglayer/agglayer#85]
## Description

The `agglayer` crate (which is the `agglayer` CLI) is the main entry point for interacting with the `agglayer`.
We should provide a good and non-breaking CLI API. To ensure that, we can create tests on this CLI by using  [cargo-insta](https://crates.io/crates/cargo-insta)

*This PR can be mentored or part of a pairing session*

## Acceptance

- [ ] Having tests that cover the `agglayer` crate using `cargo insta`

**Comment by varun-doshi** [comment]
Hey @Freyskeyd
I would like to work on this.
Would be helpful if there are any resource/starting point to go over

Thank you

**Comment by 0xaatif** [comment]
Could probably even use [`trycmd`](https://docs.rs/trycmd) or [`assert_cmd`](https://docs.rs/assert-cmd)

**Comment by varun-doshi** [comment]
Hey @Freyskeyd 
I can take this up with some more context

**Comment by Freyskeyd** [comment]
> Could probably even use [`trycmd`](https://docs.rs/trycmd) or [`assert_cmd`](https://docs.rs/assert-cmd)

Yes, both approach are valid, `assert_cmd` for CLI testing and `insta` for snapshot testing on the responses so that we don't break anything.



> Hey [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/1394604?s=64&amp;u=6c07f91eaedb3c8818aa7e03c17b93a21f767ea2&amp;v=4">@Freyskeyd](https://github.com/Freyskeyd) I can take this up with some more context

Feel free to open a PR at anytime 🙏🏼

**Comment by Freyskeyd** [comment]
Closing as the tests are currently enough
